cookbook/deployment/heroku.rst

```diff
@@ -217,6 +217,83 @@ application will respond:
 
 You should be seeing your Symfony application in your browser.
 
+Custom Compile Steps
+~~~~~~~~~~~~~~~~~~~~
+
+If you wish to execute additional custom commands during a build, you can leverage
+Heroku's `custom compile steps`_. Let's say for example you want to remove the ``dev``
+front controller from your production environment on Heroku in order to avoid a
+potential vulnerability. Adding a command to remove ``web/app_dev.php`` to Composer's
+`post-install-cmds`_ would work, but it also removes the controller in your local
+development environment on each ``composer install`` or ``composer update`` respectively.
+Instead, you can add a `custom Composer command`_ named ``compile`` to the ``scripts``
```

Comment:
Is this `compile` name mandatory? If so, we should probably say it more explicitly.